cb-wordpress-pagespeed/stackscript.sh

```diff
@@ -0,0 +1,235 @@
+#!/bin/bash
+
+set -x
+
+# <UDF name="new_username" label="Linux User Name" example="webdev" />
+# <UDF name="new_password" label="Linux User Password" />
+# <UDF name="website" label="Website Domain" example="example.com" />
+# <UDF name="website_name" label="Wordpress Website Name" />
+# <UDF name="website_username" label="Wordpress Admin Username" />
+# <UDF name="website_password" label="Wordpress Admin Password" />
+# <UDF name="website_email" label="Wordpress Admin Email" />
+# <UDF name="website_locale" label="Wordpress Locale" example="Example: de_DE, nl_NL, ru_RU" default="en_US" />
+
```

Comment:
Add logging for future troubleshooting

`# Add Logging to /var/log/stackscript.log for future troubleshooting`
`exec > >(tee /dev/ttyS0 /var/log/stackscript.log) 2>&1`

Reply:
Thanks @hmorris3293 !

I committed the change you suggested. Can you review it please?